recipe/meta.yaml

```diff
@@ -70,10 +72,12 @@ test:
     - python test_graphml.py  # [not win]
 
 about:
-  home: http://pypi.python.org/pypi/python-igraph
+  home: https://igraph.org/python/
   license: GPL
```

Comment:
```suggestion
  license: GPL-2.0-or-later
```

Reply:
Thanks, I didn't exactly know what an SPDX identifier was. It should GPLv2+ though, but I've applied that.